cli/tests/testdata/workers/close_nested_child.js

```diff
@@ -4,4 +4,5 @@ console.log("Starting the child worker");
 
 setTimeout(() => {
   console.log("The child worker survived the death of the parent!!!");
+  Deno.exit(1);
```

Comment:
Avoids the test spinning forever if it fails

Reply:
Good idea